src/widgets/helper/ChannelView.hpp

```diff
@@ -307,7 +308,7 @@ class ChannelView final : public BaseWidget
     bool isLeftMouseDown_ = false;
     bool isRightMouseDown_ = false;
     bool isDoubleClick_ = false;
-    DoubleClickSelection doubleClickSelection_;
+    Selection doubleClickSelection_{};
```

Comment:
It's already default initialized. Maybe it can be moved to `selection_`?

```suggestion
    Selection doubleClickSelection_;
```

Reply:
done a034cef8b03b6399dcc2d1f259247d875b1c0122